frontend/src/screens/settings/About.tsx

```diff
@@ -0,0 +1,54 @@
+import Loading from "src/components/Loading";
+import SettingsHeader from "src/components/SettingsHeader";
+import { useAlbyMe } from "src/hooks/useAlbyMe";
+
+import { useInfo } from "src/hooks/useInfo";
+
+export function About() {
+  const { data: info } = useInfo();
+  const { data: albyMe } = useAlbyMe();
+
+  if (!info || !albyMe) {
+    return <Loading />;
+  }
+
+  return (
+    <>
+      <SettingsHeader title="About" description="Info about your Alby Hub" />
+      <div className="grid gap-4">
+        <div className="grid gap-2">
+          <p className="font-medium text-sm">Alby Hub Version</p>
+          <p className="text-muted-foreground text-sm slashed-zero">
+            {info.version || "dev"}
+          </p>
+        </div>
+        <div className="grid gap-2">
+          <p className="font-medium text-sm">Lightning Node Backend</p>
+          <p className="text-muted-foreground text-sm slashed-zero">
+            {info.backendType}
+          </p>
+        </div>
+        <div className="grid gap-2">
+          <p className="font-medium text-sm">Nostr Relay</p>
+          <p className="text-muted-foreground text-sm slashed-zero">
+            {info.relay}
+          </p>
+        </div>
+        {info.albyAccountConnected && albyMe && (
+          <div className="grid gap-2">
+            <p className="font-medium text-sm">Connected Alby Account</p>
+            <p className="text-muted-foreground text-sm slashed-zero">
+              {albyMe.email || albyMe.lightning_address}
+            </p>
+          </div>
+        )}
+        <div className="grid gap-2">
+          <p className="font-medium text-sm">Alby Account Plan</p>
+          <p className="text-muted-foreground text-sm slashed-zero">
+            {albyMe?.subscription.plan_code || "Free"}
```

Comment:
I see `albyMe.subscription` as `{ buzz: true }` on my cloud Hub, not sure about this

Reply:
Mine is `subscription: {plan_code: "buzz_202406_month"}`. What version are you running?

We shouldn't use the `buzz` field. But I'm not sure we should show the plan code here. In https://getalby.com/subscription you should see:

Plan: Pro Cloud

Reply:
Sorry I was on 1.15, updated and now I see `buzz_202406_month` as well